src/declarations.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "*.svg";
+
+interface Choice {
+  value: string;
```

Comment:
value ma jedną z 4 możliwych wartości - świetnym typem do obsługi tego przypadku jest ENUM: https://www.typescriptlang.org/docs/handbook/enums.html

w tym przypadku będzie to:

```
enum Value {
Scissors="Scissors",
Paper="Paper",
....
....
}
```

Reply:
Następnie w kodzie zamiast pisania stringów jak 'Paper' możesz wykorzystać notację jak notacja obiektu, czyli `Value.Paper`

Reply:
i wtedy wszędzie gdzie typujesz 'value' musisz użyć tego enuma